src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts

```diff
@@ -36,6 +36,8 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
   submitInteractEdata: IInteractEventEdata;
   telemetryCdata: Array<{}>;
   instance: string;
+  isPasswordError = false;
+  passwordErrorMsg: any;
```

Comment:
as the message will be a string can we change the object type from any to string

Reply:
Done and raised PR.